index.js

```diff
@@ -229,12 +229,11 @@ class PhotoView {
         new PhotoViewManager(options).init(item)
       );
     });
-
   }
 
-  reset() {
+  reset({ animate = false } = {}) {
```

Comment:
Minor: Personally I don't think an object here is needed, but maybe it serves some purpose after all ;)